projects/lib/src/lib/portal-providers.ts

```diff
@@ -167,10 +167,10 @@ const addOptionalProviders = (
     });
   }
 
-  if (options.nodeAccessHandlingService) {
+  if (options.customNodeProcessingService) {
     providers.push({
       provide: LUIGI_NODES_ACCESS_HANDLING_SERVICE_INJECTION_TOKEN,
```

Comment:
change the const name `LUIGI_NODES_ACCESS_HANDLING_SERVICE_INJECTION_TOKEN` accordingly